src/cf-forms/src/molecules/form-fields.less

```diff
@@ -123,7 +131,7 @@
         .a-radio {
             &:focus + .a-label,
             &:checked + .a-label {
-                outline: 1px solid @input-border__focused;
+                box-shadow: 0px 0px 0px 1px @input-border__focused inset;
```

Comment:
Outline adds to the outside of the box, increasing its size by 2px in width and height. box-shadow puts the line inside the box.